docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -102,8 +98,8 @@ indices. You can modify your copy of the policy to meet your performance require
 manage resource usage.
 
 . Activate the warm phase and click **Advanced settings**.
-+
---
++ 
+-- 
```

Comment:
There is a lot of unnecessary whitespace changes in this file, can you undo those?